diff --git a/external_headers/boost/fusion/container/vector/vector10.hpp b/external_headers/boost/fusion/container/vector/vector10.hpp
index 28531b6..7241488 100644
--- a/external_headers/boost/fusion/container/vector/vector10.hpp
+++ b/external_headers/boost/fusion/container/vector/vector10.hpp
@@ -51,7 +51,7 @@ namespace boost { namespace fusion
         vector0() {}
 
         template<typename Sequence>
-        vector0(Sequence const& seq)
+        vector0(Sequence const&)
         {}
     };
 
