--- gamma.hpp	2008-08-08 01:21:09.000000000 +0200
+++ gamma.hpp.old	2008-08-08 01:20:50.000000000 +0200
@@ -1417,7 +1417,7 @@
 }
 template <class T1, class T2, class Policy>
 inline typename tools::promote_args<T1, T2>::type 
-   tgamma_ratio(T1 a, T2 b, const Policy&)
+   tgamma_ratio(T1 a, T2 b, const Policy& pol)
 {
    typedef typename tools::promote_args<T1, T2>::type result_type;
    typedef typename policies::evaluation<result_type, Policy>::type value_type;
