--- tbb_version.h.old	2008-09-29 17:50:13.000000000 +0200
+++ tbb_version.h	2008-09-29 17:50:36.000000000 +0200
@@ -33,10 +33,10 @@
 #ifndef ENDL
 #define ENDL "\n"
 #endif
-#include "version_string.tmp"
+//#include "version_string.tmp"
 
 #ifndef __TBB_VERSION_STRINGS
-#pragma message("Warning: version_string.tmp isn't generated properly by version_info.sh script!")
+//#pragma message("Warning: version_string.tmp isn't generated properly by version_info.sh script!")
 // here is an example of macros value:
 #define __TBB_VERSION_STRINGS \
 "TBB: BUILD_HOST\tUnknown\n" \
